vMenu/LanguageManager.cs

```diff
@@ -0,0 +1,45 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using MenuAPI;
+using Newtonsoft.Json;
+using CitizenFX.Core;
+using static CitizenFX.Core.UI.Screen;
+using static CitizenFX.Core.Native.API;
+using static vMenuClient.CommonFunctions;
+using static vMenuShared.ConfigManager;
+using static vMenuShared.PermissionsManager;
+
+namespace vMenuClient
+{
+    public class LanguageManager : BaseScript
+    {
+        public string Get (string originalText) {
+            
+            string transText = originalText;
+
+            if (!MainMenu.DumpedData.ContainsKey(originalText))
```

Comment:
Why is the "DumpedData" in MainMenu?
Seems best to keep all language data in one place, like here in the LanguageManager.